pkg/agent/openflow/pipeline.go

```diff
@@ -336,6 +352,9 @@ var (
 	// Endpoint, still needs to select an Endpoint, or if an Endpoint has already
 	// been selected and the selection decision needs to be learned.
 	serviceLearnRegRange = binding.Range{16, 18}
+	// serviceSnatMarkRange takes a 1-bit range of register serviceSnatReg to
+	// indicate that whether the packets of Service requires SNAT.
```

Comment:
Do you mean "indicate the packet is Service traffic that requires SNAT"? Could you also add more information about the packets (e.g. request or response, NodePort, etc.)?

Reply:
I think this can be better: `serviceSnatMarkRange takes a 1-bit range of register serviceSnatReg to mark whether the first packet of NodePort/LoadBalancer requires SNAT`